### ISSUE TITLE
Count the times user reports impediments

### ISSUE THREAD

**Issue by krzyk** [zerocracy/farm#1239]
I think we should gather statistics on the number of times a user used `@0crat wait ....` and report it as ... I don't really have a word (impedimentness?)

Similarly to verbosity, blanks, negliance a higher number of impediments is not a good thing for project.
Sometimes it is necessary, but not always.
(e.g. right now it is best to start each task with `@0crat wait` just for peace of mind).

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by g4s8** [comment]
@yegor256 please, look at this issue. What do you think?

**Comment by yegor256** [comment]
@g4s8 I agree, good idea. We also had an ticket to limit the amount of impediments any particular user may have. And an amount of impediments a project may have. Or maybe we should create such a ticket.

Say, a project may keep only 20% tasks in "on hold" state. If someone says "wait" Zerocrat says "you can do that, because 21 out of 97 tasks are already on hold"

Also, any programmer may have only 20% of his/her tasks on hold, in his/her personal agenda.

Make sense? Let's create tickets for that? I remember we had something like that, but can't find.

**Comment by g4s8** [comment]
@amihaiemil now we have 157 jobs in WBS and only 19 impediments (12%), so I think 20% is ok for the project.

**Comment by krzyk** [comment]
@g4s8 Although @amihaiemil is right about the PRs. I don't know how to solve this problem.
But there is another one, if not the reviewer and ARC is "lazy"/doesn't have time - then the reviewer has a hard limit of 10 days, and he can't do anything to avoid it, like I hade to refuse 3 PRs beause of that: https://github.com/zerocracy/farm/pull/1136, https://github.com/zerocracy/farm/pull/1137, https://github.com/zerocracy/farm/pull/1134


Maybe the speed bonus will help somehow (waiting for https://github.com/zerocracy/farm/issues/860 implementation) - This will add +5 into PRs also.

**Comment by g4s8** [comment]
@krzyk I think we can call it `delays.xml` and put a line each time user registered impediment for a job:
```xml
<delays>
  <order id="gh:test/test#1">
    <created>2018-06-26T12:50:42.387Z</created>
    <reasone>waiting for #2</reasone>
  </order>
</delays>
```

**Comment by 0crat** [comment]
Job [#1239](https://github.com/zerocracy/farm/issues/1239) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by Protagores** [comment]
@yegor256 @krzyk @g4s8 @amihaiemil I think certain issues should still be exempt from this restriction. In particular, PDD tasks which contain the word ‘impediment’ in their descriptions come to mind.
